### PR TITLE
Task 6 Get/api/articles/:article_id/comments

### PR DESCRIPTION
**Pull request by stevenwu99**
I have completed  Task 6 Get/api/articles/:article_id/comments. Please check

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -106,7 +106,49 @@ describe ('GET /api/articles/:article_id',() => {
...
+  //Task 6 GET /api/articles/:article_id/comments
+describe ('GET /api/articles/:article_id/comments',() => {
+    test ('200:should return comments by article_id',() => {
+    return request(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then (({body}) => {
+        const {comments} = body;
+        expect(comments).toBeSortedBy("created_at",{descending: true,});
+        comments.forEach((comment) => {
```
Comment (roisinoreilly) [inline __tests__/app.test.js]: please include a length check to avoid a false positive test if the array we're iterating over is empty

File: __tests__/app.test.js [hunk 2]
```diff
@@ -106,7 +106,49 @@ describe ('GET /api/articles/:article_id',() => {
...
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("votes", expect.any(Number));   
+          expect(comment).toHaveProperty("created_at");    
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("article_id", expect.any(Number));
```
Comment (roisinoreilly) [inline __tests__/app.test.js:131]: we know the article id on all of these comments will be 1 because we're only getting the comments for article 1

**Review by roisinoreilly — APPROVED** [review]
really great work Steven! I'd like you to also add in another 200 test for if the GET request is successful but there are no comments for that article, as well as a length check for your existing 200 test. Once you've sorted that PR is approved :) feel free to merge

File: __tests__/app.test.js (hunk 1 above)
Comment (stevenwu99) [reply __tests__/app.test.js]: Added a length check

File: __tests__/app.test.js (hunk 2 above)
Comment (stevenwu99) [reply __tests__/app.test.js:131]: Added  in another 200 test for if the GET request is successful but there are no comments for that article

File: __tests__/app.test.js [hunk 3]
```diff
@@ -106,7 +106,59 @@ describe ('GET /api/articles/:article_id',() => {
...
+        })
+    })
+  })
+  
+  test.only ('200:should return empty array if article_id found in article ,but not comment for this article',() => {
```
Comment (roisinoreilly) [inline __tests__/app.test.js:136]: don't forget to remove any .only tests before you push up your code

**Review by roisinoreilly — APPROVED** [review]
nicely done :) PR approved!
